rx/scheduler/eventloop/asynciothreadsafescheduler.py

```diff
@@ -130,3 +141,17 @@ def schedule_absolute(self,
 
         duetime = self.to_datetime(duetime)
         return self.schedule_relative(duetime - self.now, action, state=state)
+
+    def _is_loop_running_on_another_thread(self):
```

Comment:
can you change the name and switch the result values?
changing it to something like "self._schedule_to_self_loop" seems easier to read for me rather than "not self._is_loop_running_on_another_thread()"

Reply:
I've changed it to `_on_self_loop_or_not_running`, because if the loop is not running yet, it would also lead to deadlock.
Hope you don't mind being more clear about what function does for the cost of being longer to read?

I've also added one more test for non-relative schedule and cancel, and deduplicated test code.